README.md

```diff
@@ -26,29 +26,63 @@ pip install prefect-aws
 
 You will need to obtain AWS credentials in order to use these tasks. Refer to the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for authentication methods available.
 
-### Write and run a flow
+### Write and run a flow with AwsCredentials and S3Bucket
 
 ```python
 from prefect import flow
 from prefect_aws.s3 import s3_upload
```

Comment:
No longer need this import, but wondering if we should keep both examples, unless s3_upload is deprecated?

Reply:
We should keep both examples